share/github-backup-utils/ghe-restore-storage

```diff
@@ -144,9 +144,9 @@ for file_list in $tempdir/*.rsync; do
     --rsync-path=\"sudo -u $storage_user rsync\" \
     --files-from=$file_list \
     --size-only \
-    \"$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/storage/./\" \
-    \"$server:$GHE_REMOTE_DATA_USER_DIR/storage/\" 1>&3
-    ")
+    "$GHE_RESTORE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/storage/./" \
+    "$server:$GHE_REMOTE_DATA_USER_DIR/storage/" 1>&3
+
```

Comment:
Was there a trailing paren here?

Reply:
looks like this is from another [PR](https://github.com/github/backup-utils/pull/635/files) ... I synced from master to "fix" it.